### PR TITLE
Implement basic permission checks and Improve OIDC support.

### PR DESCRIPTION
**Pull request by jankaspar**

### CODE REVIEW

File: config/armada/config.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+development: false
```
Comment (JamesMurkin) [inline config/armada/config.yaml]: Should development be default false?

File: internal/common/startup.go [hunk 2]
```diff
@@ -44,6 +45,8 @@ func LoadConfig(config interface{}, defaultPath string, overrideConfig string) {
 	viper.SetEnvPrefix("ARMADA")
 	viper.AutomaticEnv()
 
+	fmt.Print(os.Getenv("ARMADA_DEVELOPMENT"))
```
Comment (JamesMurkin) [inline internal/common/startup.go]: Is this intended?

File: config/armada/config.yaml (hunk 1 above)
Comment (JamesMurkin) [reply config/armada/config.yaml]: Add a brief note saying to set this for local development in docs

**Comment by jankaspar** [comment]
Fixes #166.

**Pull request merged**